mergify/metadata.csv

```diff
@@ -1,2 +1,19 @@
 mergify.merge_queue_length,gauge,,,,Merge queue length,0,mergify,,
+mergify.time_to_merge.median,gauge,,second,,Median time to merge,0,mergify,,
+mergify.time_to_merge.mean,gauge,,second,,Mean time to merge,0,mergify,,
+mergify.queue_checks_outcome.PR_DEQUEUED,gauge,,,,The number of PR that have been manually removed from the queue,0,mergify,,
+mergify.queue_checks_outcome.PR_AHEAD_DEQUEUED,gauge,,,,The number of PR that have been removed from the queue because of a PR ahead of it was removed from the queue,0,mergify,,
+mergify.queue_checks_outcome.PR_AHEAD_FAILED_TO_MERGE,gauge,,,,The number of PR that have been removed from the queue because of a PR ahead of it failed to merge,0,mergify,,
+mergify.queue_checks_outcome.PR_WITH_HIGHER_PRIORITY_QUEUED,gauge,,,,The number of PR that have been removed from the queue because a PR with higher priority has been queued,0,mergify,,
+mergify.queue_checks_outcome.PR_QUEUED_TWICE,gauge,,,,The number of PR that have been removed from the queue because they have been queued twice,0,mergify,,
+mergify.queue_checks_outcome.SPECULATIVE_CHECK_NUMBER_REDUCED,gauge,,,,The number of PR that have been removed from the queue because the number of speculative check in the config was changed,0,mergify,,
+mergify.queue_checks_outcome.CHECKS_TIMEOUT,gauge,,,,The number of PR that have been removed from the queue because the speculative checks have timed out,0,mergify,,
+mergify.queue_checks_outcome.CHECKS_FAILED,gauge,,,,The number of PR that have been removed from the queue because the speculative checks have failed,0,mergify,,
+mergify.queue_checks_outcome.QUEUE_RULE_MISSING,gauge,,,,The number of PR that have been removed from the queue because the queue rule that was used to queue the PR has been removed from the config,0,mergify,,
+mergify.queue_checks_outcome.UNEXPECTED_QUEUE_CHANGE,gauge,,,,The number of PR that have been removed from the queue because a user made an operation on the queued pull request,0,mergify,,
+mergify.queue_checks_outcome.PR_FROZEN_NO_CASCADING,gauge,,,,The number of PR that have been removed from the queue because it was frozen by a freeze with no cascading effect,0,mergify,,
+mergify.queue_checks_outcome.TARGET_BRANCH_CHANGED,gauge,,,,The number of PR that have been removed from the queue because the PR's target branch was changed,0,mergify,,
+mergify.queue_checks_outcome.TARGET_BRANCH_MISSING,gauge,,,,The number of PR that have been removed from the queue because the PR's target branch does not exist anymore,0,mergify,,
+mergify.queue_checks_outcome.PR_UNEXPECTEDLY_FAILED_TO_MERGE,gauge,,,,The number of PR that have been removed from the queue because it unexpectedly failed to get merged,0,mergify,,
+mergify.queue_checks_outcome.BATCH_MAX_FAILURE_RESOLUTION_ATTEMPTS,gauge,,,,The number of PR that have been removed from the queue because the maximum batch failure resolution attempts has been reached,0,mergify,,
```

Comment:
```suggestion
mergify.queue_checks_outcome.PR_WITH_HIGHER_PRIORITY_QUEUED,gauge,,,,The number of PRs that have been removed from the queue because a PR with higher priority has been queued,0,mergify,,
mergify.queue_checks_outcome.PR_QUEUED_TWICE,gauge,,,,The number of PRs that have been removed from the queue because they have been queued twice,0,mergify,,
mergify.queue_checks_outcome.SPECULATIVE_CHECK_NUMBER_REDUCED,gauge,,,,The number of PRs that have been removed from the queue because the number of speculative checks in the config was changed,0,mergify,,
mergify.queue_checks_outcome.CHECKS_TIMEOUT,gauge,,,,The number of PRs that have been removed from the queue because the speculative checks have timed out,0,mergify,,
mergify.queue_checks_outcome.CHECKS_FAILED,gauge,,,,The number of PRs that have been removed from the queue because the speculative checks have failed,0,mergify,,
mergify.queue_checks_outcome.QUEUE_RULE_MISSING,gauge,,,,The number of PRs that have been removed from the queue because the queue rule that was used to queue the PR has been removed from the config,0,mergify,,
mergify.queue_checks_outcome.UNEXPECTED_QUEUE_CHANGE,gauge,,,,The number of PRs that have been removed from the queue because a user made an operation on the queued pull request,0,mergify,,
mergify.queue_checks_outcome.PR_FROZEN_NO_CASCADING,gauge,,,,The number of PRs that have been removed from the queue because the PR was frozen by a freeze with no cascading effect,0,mergify,,
mergify.queue_checks_outcome.TARGET_BRANCH_CHANGED,gauge,,,,The number of PRs that have been removed from the queue because the PR's target branch was changed,0,mergify,,
mergify.queue_checks_outcome.TARGET_BRANCH_MISSING,gauge,,,,The number of PRs that have been removed from the queue because the PR's target branch does not exist anymore,0,mergify,,
mergify.queue_checks_outcome.PR_UNEXPECTEDLY_FAILED_TO_MERGE,gauge,,,,The number of PRs that have been removed from the queue because the PR unexpectedly failed to get merged,0,mergify,,
mergify.queue_checks_outcome.BATCH_MAX_FAILURE_RESOLUTION_ATTEMPTS,gauge,,,,The number of PRs that have been removed from the queue because the maximum batch failure resolution attempts have been reached,0,mergify,,
```

Reply:
Thanks for the review. Made the modifications, i just replaced the last two `the PR` suggestion you made with `they` instead, sounded more fitting imo.